apps/examples/lcd_test/example_lcd.c

```diff
@@ -363,6 +425,25 @@ static void test_fps(void)
 	}
 }
 
+bool is_valid_power(char *power)
+{
+	int power_val_size = strlen(power);
+
+	if (power_val_size < 1 || power_val_size > 3) {		/* Length of Power val should be 1, 2, or 3 */
+		return false;
+	}
+	for (int i = 0; i < power_val_size; i++) {
+		if (!isdigit(power[i])) {	/* If not a digit */
+			return false;
+		}
+	}
+	int power_val = atoi(power);
+	if (power_val < 0 || power_val > 100) {	/* Value of power should be in range [0, 100] */
+		return false;
+	}
+	return true;
```

Comment:
the lcd_setpower is already returning error if power val input is over than CONFIG_LCD_MAXPOWER.
so, we don't need to check this value here.